### PR TITLE
l5: tunspace & single-file

### PR DESCRIPTION
**Pull request by pktpls**

### CODE REVIEW

File: locations/l5.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+longitude: 13.42552
+contact_nickname: L5 Kollektiv
+contacts:
+  - noc@stadtfunk.net
+
+location__ssh_keys__to_merge:
+  - comment: Hener
+    key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEST9QsXtphN7BYb5p9FhxZTxvoWkICfRWC54SN/QHII
+  - comment: Lino
+    key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOaljQHr4Nj6veeDi51BZjy/a3WXP
```
Comment (Noki) [inline locations/l5.yml:14]: this can be dropped, both keys are in the default list

File: locations/l5.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    wireless_profile: freifunk_default
+    host__packages__to_merge:
+      - -tunnelmanager
+      - tunspace
+      - wireguard-tools
+
+  - hostname: l5-nf-o
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    # TODO: mac_override
```
Comment (Noki) [inline locations/l5.yml:30]: cat /sys/firmware/mikrotik/hard_config/mac_base

File: locations/l5.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  - hostname: l5-nf-w
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    # TODO: mac_override
```
Comment (Noki) [inline locations/l5.yml:35]: cat /sys/firmware/mikrotik/hard_config/mac_base

**Review by Noki — CHANGES_REQUESTED** [review]

**Comment by pktpls** [comment]
Nah this PR is purely for single-file and tunspace conversion, the location maintainers can take care of other changes :+1:

**Review by Noki — APPROVED** [review]

**Pull request merged**